### PR TITLE
Skeleton Structure

### PR DESCRIPTION
**Pull request by samgdotson**
This PR adds the basic structure of the ``pygenesys`` package. A minimum working framework for what will become a larger tool. Included is:
* circleci configuration file
* ``pygenesys`` folder
* ``tests`` folder 
* one test 
* ``technology`` file, with a placeholder function (s.t. I could create a basic test and run it). 
* requirements file
* python setup file (s.t. the module can be installed as a python package). 
* License
* gitignore
* README

The package installs successfully with 
```bash
python setup.py install
```
There are no dependencies at the moment. They will be added to ``requirements.txt`` as necessary. 
The tests also pass. Circleci was having trouble establishing a config file from the dashboard, so I did it manually.

If you are testing this code, I recommend creating a new conda environment

```bash
conda create --name pygen-test python=3.7
conda activate pygen-test
cd pygenesys
python setup.py install
```

Thanks!

### CODE REVIEW

**Review by nsryan2 — CHANGES_REQUESTED** [review]
Running the setup, per the readme, it was overall successful. I did notice, however, that it looked for numpy, so that should probably be in the requirements.txt file.

File: README.md [hunk 1]
```diff
@@ -1,2 +1,29 @@
-# pygenesys
-Python for Generating Energy Systems
+# Python for Generating Energy Systems (PyGenESys)
+
+This package can be used to generate input files for the [Temoa](https://github.com/temoaproject/temoa) modelling tool. Transparency and repeatability are integral to the design motivations for Temoa. PyGenesys takes this concept
+even further by reducing the lead time and programming background required to
+run Temoa.
+PyGenesys offers technology models with cost and production time series data.
+The basic model is based on the state of Illinois and its flagship university, the University of Illinois at Urbana-Champaign.
```
Comment (nsryan2) [inline README.md:7]:
```suggestion
The model is based on the state of Illinois and its flagship university, the University of Illinois at Urbana-Champaign.
```

**Review by nsryan2 — APPROVED** [review]
Ok, I think this should change my previous review. I had a minor edit to the readme as well.

File: pygenesys/version.py [hunk 2]
```diff
@@ -0,0 +1,68 @@
+from __future__ import absolute_import, division, print_function
+from os.path import join as pjoin
+
+# Format expected by setup.py and doc/source/conf.py: string of form "X.Y.Z"
+_version_major = 0
+_version_minor = 1
+_version_micro = ''  # use '' for first of series, number for 1 and above
+_version_extra = 'dev'
```
[the remark this answers is not in the document]
Comment (samgdotson) [reply pygenesys/version.py:8]: I'm unsure of the purpose of this suggestion. The style is from [shablona](https://github.com/uwescience/shablona). I can't find this additionaly zero anywhere in software versioning best practices. I'll consider this handled unless there is a compelling reason to change it.

File: README.md (hunk 1 above)
Comment (samgdotson) [reply README.md:7]: ``pygenesys`` is a model design tool and thus isn't specific to a particular region or system. Perhaps "test" model would have been better. But that's still inaccurate. The README will become clearer as the code base develops.

**Comment by samgdotson** [comment]
@nsryan2 if you think it's fine and everything works you can go ahead and merge it.

File: pygenesys/version.py (hunk 2 above)
[the remark this answers is not in the document]
Comment (nsryan2) [reply pygenesys/version.py:8]: No compelling reason other than that there was a warning raised where it changed it to say dev0 automatically so I found where it came from in the code itself.

File: README.md [hunk 3]
```diff
@@ -1,2 +1,18 @@
-# pygenesys
-Python for Generating Energy Systems
+# Python for Generating Energy Systems (PyGenESys)
+
+This package can be used to generate input files for the [Temoa](https://github.com/temoaproject/temoa) modelling tool. Transparency and repeatability are integral to the design motivations for Temoa. PyGenesys takes this concept
```
Comment (npanczyk) [inline README.md:3]: U.S. spelling is usually "modeling" but "modelling" is used everywhere else, just figured I'd point it out

**Review by npanczyk — COMMENTED** [review]
this looks good to me! looking forward to trying this with hydrogen

**Pull request merged**